src/ResourceManager/Resources/Commands.Resources/ChangeLog.md

```diff
@@ -26,6 +26,7 @@
 * Support managed identity on policy assignments.
 * Parameters with default values are no longer requred when assigning a policy with `New-AzureRmPolicyAssignment`
 * Add new cmdlet Get-AzureRmPolicyAlias for retrieving policy aliases
+* Update cmdlet Test-AzureRmNetworkWatcherConnectivity, pass the protocol value to backend.
```

Comment:
@junyezhu this needs to go in the [Network change log](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/ChangeLog.md) under the `## Current Release` header